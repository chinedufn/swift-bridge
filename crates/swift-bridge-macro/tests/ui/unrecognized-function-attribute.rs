//! # To Run
//! cargo test -p swift-bridge-macro -- ui trybuild=unrecognized-function-attribute.rs

#[swift_bridge::bridge]
mod ffi {
    extern "Rust" {
        #[swift_bridge(InvalidAttribute)]
        fn some_function();
    }
}

pub struct SomeType;

fn main() {}
