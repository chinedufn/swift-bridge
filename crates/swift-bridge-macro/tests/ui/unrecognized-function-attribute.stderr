error: Unrecognized attribute "InvalidAttribute".
 --> tests/ui/unrecognized-function-attribute.rs:7:24
  |
7 |         #[swift_bridge(InvalidAttribute)]
  |                        ^^^^^^^^^^^^^^^^
